### PR TITLE
Release 2.1.2

### PR DESCRIPTION
**Pull request by niik**
## Overview

Preparation PR for the 2.1.2 production release which we're planning on releasing today (Aug 21).

## Tasks

- [x] Prepare changelog
- [x] Update version number
- [x] Sanity check last beta release for new and exciting crashes
- [x] Release it

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,27 @@
 {
   "releases": {
+    "2.1.2": [
+      "[Added] Syntax highlighting support for 20 more programming languages - #7217. Thanks @KennethSweezy!",
+      "[Added] Kitty shell support for macOS - #5162",
+      "[Added] Atom editor support on Windows for beta and nightly channels - #8010. Thanks @Rexogamer!",
+      "[Added] WSL Support - #7821 #7891. Thanks @ADustyOldMuffin @say25!",
```
Comment (billygriffin) [inline changelog.json]: I think this is just enabled for beta https://github.com/desktop/desktop/pull/7922
```suggestion
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+      "[Fixed] File checkboxes and line selection in diffs are not disabled when committing - #3814 #5934. Thanks @HashimotoYT!",
+      "[Fixed] Small images are scaled down too much in two-up image rendering - #7520",
+      "[Fixed] Manual conflict resolution for binary files always chooses same version - #8059",
+      "[Fixed] Disable some unavailable actions while committing - #3814 #5934. Thanks @HashimotoYT!",
```
Comment (billygriffin) [inline changelog.json]: Looks like a duplicate of 3 lines above, and that note is more clear.
```suggestion
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+      "[Fixed] Branch pruner errors on \"orphaned\" branches - #7983",
+      "[Fixed] Prevent CodeMirror search plugin from stealing registered global keyboard shortcuts - #8068",
+      "[Fixed] Update embedded Git on Windows to address security vulnerability - #8101, #8133",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Update embedded Git on Windows to remove erroneous errors - #8101, #8133",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+      "[Fixed] Menu state not updated for macOS after performing some actions - #8055",
+      "[Fixed] Manual conflict resolution for binary files - #8059",
```
Comment (billygriffin) [inline changelog.json]: Duplicate of one above.
```suggestion
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+      "[Fixed] Error message could appear outside the boundary of its container - #7988",
+      "[Fixed] Improved error handling when trying to rebase a ref that doesn't exist - #7881",
+      "[Improved] Upgrade to Electron 5 - #6888",
```
Comment (billygriffin) [inline changelog.json]: Pretty sure we typically don't include purely infrastructural things?
```suggestion
```

File: changelog.json (hunk 3 above)
Comment (niik) [reply changelog.json]: Ah, you're right. And we should drop the #8101 link too.

File: changelog.json (hunk 5 above)
Comment (niik) [reply changelog.json]: Yeah that makes sense. I copy pasted it from the beta changelog entries where I supposed it makes more sense to include it.

**Pull request merged**